docker-compose.yml

```
@@ -0,0 +1,7 @@
+jekyll:
+    image: jekyll/jekyll:latest
+    command: jekyll serve --force_polling
```

그대로 빌드할 경우 제 환경에서는 
```docker-compose up
Starting convex-optimization-for-allgithubio_jekyll_1 ... done
Attaching to convex-optimization-for-allgithubio_jekyll_1
jekyll_1  | /usr/local/lib/ruby/site_ruby/2.7.0/rubygems.rb:281:in `find_spec_for_exe': Could not find 'bundler' (1.17.2) required by your /srv/jekyll/Gemfile.lock. (Gem::GemNotFoundException)
jekyll_1  | To update to the latest version installed on your system, run `bundle update --bundler`.
jekyll_1  | To install the missing version, run `gem install bundler:1.17.2`
jekyll_1  |     from /usr/local/lib/ruby/site_ruby/2.7.0/rubygems.rb:300:in `activate_bin_path'
jekyll_1  |     from /usr/local/bin/bundle:23:in `<main>'
convex-optimization-for-allgithubio_jekyll_1 exited with code 1
```
의 로그와 함께 실패합니다.

로그를 보면 패키지가 없다고 하여

```suggestion
    command: >
        bash -c "gem install bundler:1.17.2
        && jekyll serve --force_polling"
```
패키지 설치 명령을 추가하면 문제 없이 작동합니다.

---

혹은 이미지 업데이트 이슈를 고려하여 저희가 이미지를 만들어서 활용해도 좋을 것 같습니다.

간단하게 이미지 구워서 다시 테스트해보니 잘 되네요 ㅎㅎ

---

@hunhoon21 
1. `docker-compose` 명령어로 실행하신건가요?
2. `docker-compose.yml`과 동일한 경로상에서 실행하셨나요?

---

> @hunhoon21
> 
> 1. `docker-compose` 명령어로 실행하신건가요?
> 2. `docker-compose.yml`과 동일한 경로상에서 실행하셨나요?

```
convex) ➜  convex-optimization-for-all.github.io git:(setting/add_docker_compose)  docker-compose up    
Starting convex-optimization-for-allgithubio_jekyll_1 ... done
Attaching to convex-optimization-for-allgithubio_jekyll_1
jekyll_1  | /usr/local/lib/ruby/site_ruby/2.7.0/rubygems.rb:281:in `find_spec_for_exe': Could not find 'bundler' (1.17.2) required by your /srv/jekyll/Gemfile.lock. (Gem::GemNotFoundException)
jekyll_1  | To update to the latest version installed on your system, run `bundle update --bundler`.
jekyll_1  | To install the missing version, run `gem install bundler:1.17.2`
jekyll_1  |     from /usr/local/lib/ruby/site_ruby/2.7.0/rubygems.rb:300:in `activate_bin_path'
jekyll_1  |     from /usr/local/bin/bundle:23:in `<main>'
convex-optimization-for-allgithubio_jekyll_1 exited with code 1
(convex) ➜  convex-optimization-for-all.github.io git:(setting/add_docker_compose)  cat docker-compose.yml            
jekyll:
    image: jekyll/jekyll:latest
    command: jekyll serve --force_polling
    ports:
        - 4000:4000
    volumes:
        - .:/srv/jekyll
```

네~ 맞습니다!

---

수정사항 반영했습니다.